bower.json

```diff
@@ -25,7 +25,7 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "polymer/polymer#~0.5.5",
+    "polymer": "polymer/polymer#~1.0.0",
```

Comment:
This is not a correct selector, because it will match version range `>= 1.0.0` and `< 1.1.0`.

What you mean is `"polymer/polymer#^1.0.0"`. This means `>= 1.0.0`